### DISCUSSION TITLE
separate universes for outputs

### DISCUSSION THREAD

**Discussion by perigalacticon** [forkineye/ESPixelStick#594] [General]
My props are on separate universes in XLights.  The universe numbers are not necessarily consecutive order by start channel number in XLights.  Is there a way to have ESPixelstick assign the data by universe number only to the different outputs? ie universe 1 on output 1, universe 2 on output 2 etc...

**Comment by MartinMueller2003** [comment]
There was a deliberate design decision made to not drag the inefficient E1.31 data distribution protocol into the basic ESP V4 design. xLights does not require you to ,make such artificial assignments and such assignments waste significant WiFi resources, reducing the number of channels WiFi can support.
